pmix_governance.tex

```diff
@@ -245,7 +256,7 @@ \subsubsection{Modification of Governance Rules}\label{modification-of-governanc
 all discussion over the proposal. When ready for review, the author(s)
 shall add the ``Proposed'' tag to the PR, thereby requesting that the
 matter be placed on the agenda of the next quarterly meeting (subject to
-the one month deadline). The \textit{Straw Poll} mechanism described
+the agenda finalization deadline). The \textit{Straw Poll} mechanism described

```

Comment:
Is this really intended to be the "agenda finalization" deadline, which is two weeks before the meeting ("The agenda will be finalized two weeks before the quarterly meeting")? Shouldn't this be the deadline for submissions to the agenda? Same for the rest of the entries?

Reply:
I think agenda finalization is correct here. The community can ask for something to be added to the agenda between the post of the draft agenda at 4 weeks and the finalization at 2 weeks. That makes the 4 week deadline a soft deadline.

Reply:
There's nothing wrong with allowing a little human judgement into the process...we don't have to be rigid as robots here.

Reply:
That's fine, but I am just pointing out that it contradicts the text above then, where it says that agenda items must be submitted 4 weeks ahead of time. Knowing how things work, it will, however, mean that you (over time) will get all agenda items 14 days before the meeting and the draft agenda is pointless. We could just state a two week deadline for items and then let the chairs handle the rest with judgement (as we do in MPI).

Reply:
Per conversation during the 3Q 2020 ASC meeting:
 * The sense of the group was that we intended this to be the agenda finalization deadline to allow for missed items to be added between the 4 week and 2 week time periods.